cheroot/connections.py

```diff
@@ -99,11 +99,12 @@ def expire(self):
         # find any connections still registered with the selector
         # that have not been active recently enough.
         threshold = time.time() - self.server.timeout
-        timed_out_connections = (
+        timed_out_connections = [
             (sock_fd, conn)
-            for _, (_, sock_fd, _, conn) in self._selector.get_map().items()
+            for _, (_, sock_fd, _, conn)
+            in self._selector.get_map().items()
             if conn != self.server and conn.last_used < threshold
-        )
+        ]
```

Comment:
Can we please revert this change? Since the code below stops modifying the map, it should be no longer necessary.

Reply:
Oh, it's a separate function. I take it back, then.